Alignment/HIPAlignmentAlgorithm/python/ALCARECOTkAlCosmicsInCollisionsTrackSelection_cff_py.txt

```diff
@@ -0,0 +1,79 @@
+process.TrackerTrackHitFilter.replaceWithInactiveHits = True
+process.TrackerTrackHitFilter.stripAllInvalidHits = False
+process.TrackerTrackHitFilter.rejectBadStoNHits = True
+#process.TrackerTrackHitFilter.StoNcommands = cms.vstring("ALL 12.0")  #deco mode
+process.TrackerTrackHitFilter.StoNcommands = cms.vstring("ALL 18.0")   #peak mode
+process.TrackerTrackHitFilter.rejectLowAngleHits= True
+process.TrackerTrackHitFilter.TrackAngleCut= 0.1 # use 0.1 for 0T 
+#process.TrackerTrackHitFilter.TrackAngleCut= 0.35 # use 0.35 for 3.8T 
+process.TrackerTrackHitFilter.usePixelQualityFlag= True
+
+
+# track producer to be run after the alignment hit filter
+import RecoTracker.TrackProducer.CTFFinalFitWithMaterialP5_cff
+process.ctfProducerCustomised = RecoTracker.TrackProducer.CTFFinalFitWithMaterialP5_cff.ctfWithMaterialTracksCosmics.clone()
+process.ctfProducerCustomised.src = 'TrackerTrackHitFilter'
+process.ctfProducerCustomised.TTRHBuilder = 'WithAngleAndTemplate'
+#process.ctfProducerCustomised.TrajectoryInEvent = False
+process.ctfProducerCustomised.NavigationSchool = ''
+
+
+#process.TrackRefitter2 = RecoTracker.TrackProducer.TrackRefitterP5_cfi.TrackRefitterP5.clone()
+#process.TrackRefitter2.src = 'AlignmentTrackSelector'
+#process.TrackRefitter2.TrajectoryInEvent = True
+#process.TrackRefitter2.TTRHBuilder = "WithAngleAndTemplate"
+#process.TrackRefitter2.NavigationSchool = ''
+
+process.TrackRefitter2 = process.TrackRefitter1.clone()
+process.TrackRefitter2.src = 'AlignmentTrackSelector'
+
+
+# track selection for sample in filename...
+process.AlignmentTrackSelector.src = 'ctfProducerCustomised'
+process.AlignmentTrackSelector.applyBasicCuts = True
+process.AlignmentTrackSelector.pMin = 4
+process.AlignmentTrackSelector.ptMin = 0
+process.AlignmentTrackSelector.etaMin = -999.
+process.AlignmentTrackSelector.etaMax = 999.
+process.AlignmentTrackSelector.d0Min = -50.
+process.AlignmentTrackSelector.d0Max = 50.
+process.AlignmentTrackSelector.nHitMin = 8
+process.AlignmentTrackSelector.nHitMin2D = 2 
+process.AlignmentTrackSelector.chi2nMax = 9999.
+process.AlignmentTrackSelector.applyMultiplicityFilter = True# False
+process.AlignmentTrackSelector.maxMultiplicity = 1
+process.AlignmentTrackSelector.applyNHighestPt = False
+process.AlignmentTrackSelector.nHighestPt = 1
+process.AlignmentTrackSelector.seedOnlyFrom = 0
+process.AlignmentTrackSelector.applyIsolationCut = False
+process.AlignmentTrackSelector.minHitIsolation = 0.8
+process.AlignmentTrackSelector.applyChargeCheck = False # since no S/N cut is applied
+process.AlignmentTrackSelector.minHitChargeStrip = 50.
+
+# Reject outliers
+## include  "TrackingTools/TrackFitters/data/RungeKuttaKFFittingSmootherESProducer.cfi"
+#process.KFFittingSmootherWithOutliersRejectionAndRK.EstimateCut=18.0
+#process.KFFittingSmootherWithOutliersRejectionAndRK.MinNumberOfHits=5
+process.FittingSmootherRKP5.EstimateCut=18.0
+process.FittingSmootherRKP5.MinNumberOfHits=5
+
```

Comment:
@usarica you could consider using the [common track selection and refitting sequence](https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/Alignment/CommonAlignment/python/tools/trackselectionRefitting.py)